app/views/reservations/edit.html.haml

```diff
@@ -25,6 +25,13 @@
 
   = render "reservation_fields", f: f
 
+  .well
+    .container
+      .row
+        .span
+          %h4= OrderDetail.model_name.human
```

Comment:
There's a field `admin_note` on `Reservation` and a field `note` on `OrderDetail`--since this is `reserverations/edit` and not `order_details/edit`, I wasn't sure if this would be confusing with no additional clarification. I can see it not being a problem though, users prob have more context/understanding?